### ISSUE TITLE
Unknown 'map-edge color' beyond specified visibilitylimits

### ISSUE THREAD

**Issue by Cryptite** [webbukkit/dynmap#1989]
I have the following map rendered. I'm aware of how to define a custom texture pack to render the map with (it's what I used here to get a specific water color). I'm also aware of the using background as a definition of background color, but what I can't seem to gather is what this brown edge that it's rendering _beyond_ the specified visibility-limits. Anybody know?

![image](https://cloud.githubusercontent.com/assets/399721/17646453/b1d432d2-618f-11e6-9a66-cbf675e85e63.png)

**Comment by A248** [comment]
That's **hidestyle** from your worlds.txt. hidestyle applies to all rendered chunks outside of your visibility limits. By default it is set to stone, for the texture of stone. Other options are ocean and air. Air = same texture as un-rendered, ocean = texture of a plain ocean.

Go to [this page](https://github.com/webbukkit/dynmap/wiki/World-and-template-settings), press CTRL + F, and type "hidestyle." You will find official information about the configuration option.

And yes, I know this is a necro post; however, I came across this issue when I encountered the same situation. I myself wasn't sure what that texture was. Finding the answer, I proceeded to add it here so others wondering about this might know.
